tests/test-cli-output.lua

```diff
@@ -0,0 +1,26 @@
+-- This can hardly be called a proper test, but it should help refactoring the CLI functionality (no logic should be changed)
+
+local commands = {
+	["luvi ."] = "lit.txt",
+	["luvi . -- help"] = "lit-help.txt",
+	["luvi . -- version"] = "lit-version.txt",
+	["luvi . -- ls"] = "lit-ls.txt",
+	["luvi . -- asdf"] = "invalid-command.txt",

```

Comment:
These tests seem very brittle, e.g. they would have to be updated with every new version. Not sure if they're worth committing to the repository.

Reply:
Agreed, and I'm not at all happy with that (but it was better than not having anything to fall back on).

Unfortunately I can't think of a better way to test the CLI without digging into the internals, which would ideally have separate (unit) tests. Do you have any suggestions or should I just completely remove it?

Reply:
I think removing them is fine. They were useful for your particular use case but I'm not sure they'd be too useful to keep around.